packages/launchpad/src/setup/FrameworkOptionsFooter.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div class="flex bg-gray-50 h-48px py-12px px-17px text-gray-800 items-center">
+    <div>
+      <Icon
+        name="general-grid-2x2"
+        size="16"
+        stroke-color="gray-500"
+        fill-color="gray-100"
+      />
+    </div>
+    <div class="ml-15px">
+      <ExternalLink
+        href="https://on.cypress.io/component-integrations"
```

Comment:
Nice - is there a services PR with this? It should go to https://docs.cypress.io/guides/component-testing/third-party-definitions

Reply:
https://github.com/cypress-io/cypress-services/pull/5366

Any idea about the UTM params we need?

Reply:
Added UTM params